lib/src/signed_video_h26x_common.c

```diff
@@ -1261,9 +1251,9 @@ signed_video_reset(signed_video_t *self)
     SV_THROW(legacy_sv_reset(self->legacy_sv));
     self->signing_started = false;
     self->sei_generation_enabled = false;
+    self->validation_flags.has_lost_sei = false;
```

Comment:
Not needed since the flags are re-initialized below through `validation_flags_init(...)`